website/scripts/should-build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
I like to avoid bash specifically whenever possible. Since this doesn't need any bash-only functions, it can use /bin/sh instead, for portability.

You'll need to change the tests to `[ "$VERCEL_GIT_COMMIT_REF" = "..." ]`